msvc/sdk-build.targets

```diff
@@ -38,7 +38,7 @@
       <GetPackageContentsDependsOn>$(GetPackageContentsDependsOn);$(ComputeLinkInputsTargets)</GetPackageContentsDependsOn>
   </PropertyGroup>
 
-  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents"  DependsOnTargets="$(GetPackageContentsDependsOn)" Condition="'$(ConfigurationType)' == 'DynamicLibrary' or '$(ConfigurationType)' == 'Application'">
+  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" DependsOnTargets="$(GetPackageContentsDependsOn);_GenerateProjectPriFileCore" Condition="'$(ConfigurationType)' == 'DynamicLibrary' or '$(ConfigurationType)' == 'Application'">
```

Comment:
>_GenerateProjectPriFileCore [](start = 116, length = 27)

This should probably be in a conditioned property group to += it onto GetPackageContentsDependsOn based on if its an AppContainerApplication project. 

Since this is being added to build.sln's common targets, you should probably consider adding it to the Packaging Package's default *production* targets so that the same things happen for middleware consumers.